test/tests/mortar/displaced-gap-conductance-2d-bnd-coupling/tests

```diff
@@ -0,0 +1,20 @@
+[Tests]
+  design = 'Constraints/index.md'
+  issues = '#13080'
+  [gap-conductance-bnd-aux-kernel]
+    type = Exodiff
+    input = 'gap-conductance-bnd-aux-kernel.i'
+    exodiff = 'gap-conductance-bnd-aux-kernel_out.e'
+    custom_cmp = 'exodiff.cmp'
+    mesh_mode = 'REPLICATED'
```

Comment:
@lindsayad Mortar does not work with distributed mesh yet?

Reply:
No it does not because I either need to be able to have mesh generators request ghosting or I need to put capability for push/pull element data in the LowerDBlocksFromSidesetGenerator. It should not be hard to do but I have not had time yet